cratedb_django/models/model.py

```diff
@@ -16,12 +20,12 @@ def __new__(cls, name, bases, attrs, **kwargs):
 
         try:
             meta = attrs["Meta"]
-            for crate_attr in CRATE_META_OPTIONS:
-                attr_name = crate_attr[0]
-                if hasattr(meta, attr_name):
-                    crate_attrs[attr_name] = meta.__dict__[attr_name]
-                    delattr(meta, attr_name)
-
+            for key, default_value in CRATE_META_OPTIONS.items():
+                if key in meta.__dict__:
+                    crate_attrs[key] = meta.__dict__[key]
+                    delattr(meta, key)
+                else:
+                    crate_attrs[key] = default_value
```

Comment:
Just golfing... Would that possibly also work equally well?
```suggestion
            for key, default_value in CRATE_META_OPTIONS.items():
                crate_attrs.set(key, meta.pop(key, default_value))
```